common/map_file_ros2/config/parameters.yaml

```diff
@@ -0,0 +1,9 @@
+update_rate: 0
```

Comment:
Should this package have a param file? In the ros1 implementation the values for the params in these nodes comes from carma. The same approach is used here, the parameter values are directly passed in carma-config and platform, 
having a config file creates a false sense that the parameter values can be controlled from here.

Also there are 4 nodes within this package with different launch files. If we do want to keep the param yml, we need to add params for all the nodes.
The ones specified here are only for the points_map_loader node.

Reply:
I don't think we need it if we are not using it

Reply:
Removed this params file. Parameters will need to be passed as arguments when launching the node